lib/metanorma/cli/commands/config.rb

```diff
@@ -0,0 +1,122 @@
+require "pathname"
+
+require "metanorma/cli/stringify_all_keys"
+
+module Metanorma
+  module Cli
+    module Commands
+      class Config < Thor
+        class_option :global, aliases: "-g", type: :boolean, default: false, desc: "Use global config"
+
+        desc "get NAME", "Get config value"
+        def get(name = nil)
+          config_path = Metanorma::Cli.config_path(options.global)
+          config = load_config(config_path)
```

Comment:
Looks like we are doing the similar things in more than one places what do you think about extracting it into a private method maybe, something like

```ruby
def config_path(global_flag)
  load_config(Metanorma::Cli.config_path(global_flag)
end

# usages could be config_path(options.global)
```

Reply:
Good catch